docs/contributing/how-to-contribute.md

```diff
@@ -101,7 +101,7 @@ In order to accept your pull request, we need you to submit a CLA. You only need
 
 ### Development Workflow
 
-After cloning React, run `npm install` to fetch its dependencies.  
```

Comment:
I wasn't aware whitespace was meaningful in that context easier, thanks @gaearon